consensus/safety-rules/src/tests/suite.rs

```diff
@@ -660,7 +666,6 @@ fn test_validator_not_in_set(safety_rules: &Callback) {
         Some(next_epoch_state),
         key.as_ref(),
     );
-    safety_rules.construct_and_sign_vote(&a2).unwrap();
```

Comment:
Just some random cleanup?

Reply:
no, this fails after we added block.verify_well_formed check